packages/@lwc/integration-karma/test/shadow-dom/multiple-templates/index.spec.js

```diff
@@ -41,63 +41,66 @@ if (process.env.NATIVE_SHADOW) {
     });
 }
 
-describe('multiple stylesheets rendered in same component', () => {
-    it('works when first template has no style but second template does', () => {
-        const element = createElement('x-multi-no-style-in-first', { is: MultiNoStyleInFirst });
-        document.body.appendChild(element);
-        return Promise.resolve()
-            .then(() => {
-                expect(getComputedStyle(element.shadowRoot.querySelector('.red')).color).toEqual(
-                    'rgb(0, 0, 0)'
-                );
-                expect(getComputedStyle(element.shadowRoot.querySelector('.green')).color).toEqual(
-                    'rgb(0, 0, 0)'
-                );
-                expect(getComputedStyle(element).marginLeft).toEqual('0px');
-                element.next();
-                return new Promise((resolve) => requestAnimationFrame(() => resolve()));
-            })
-            .then(() => {
-                expect(getComputedStyle(element.shadowRoot.querySelector('.red')).color).toEqual(
-                    'rgb(255, 0, 0)'
-                );
-                expect(getComputedStyle(element.shadowRoot.querySelector('.green')).color).toEqual(
-                    'rgb(0, 128, 0)'
-                );
-                expect(getComputedStyle(element).marginLeft).toEqual('5px');
-                element.next();
-                return new Promise((resolve) => requestAnimationFrame(() => resolve()));
-            })
-            .then(() => {
-                if (process.env.NATIVE_SHADOW) {
-                    // TODO [#2466]: In native shadow, stylesheets are not removed from the DOM
+// Edge 18 renders styles incorrectly in this test
```

Comment:
This is broken in Edge 18, but was not broken in IE11? Or was this test never run for IE11?

Reply:
Yeah good point, this is broken in Edge 18 but not IE11. Must have been a regression in legacy Edge or something.